test.sh

```diff
@@ -152,4 +152,7 @@ GO111MODULE=on test_project project-v3 3-alpha
 GO111MODULE=on test_project project-v3-multigroup 3-alpha
 GO111MODULE=on test_project project-v3-addon 3-alpha
 
+# test script that setup envtest
+./scripts/setup_envtest_bins.sh v1.18.2 v3.4.3
```

Comment:
I'm struggling to understand why we are installing the envtest binaries right before exiting the test.

Reply:
It just for we ensure that the scripts are working in both platform. 
The idea is to remove it in the follow-up https://github.com/kubernetes-sigs/kubebuilder/pull/1600 where they will be called for each testadata sample via the makefile target instead.